### DISCUSSION TITLE
after Update to 1.2.2 Installation failed with "Advanced: elevate permissions " checked

### DISCUSSION THREAD

**Discussion by scrichab** [wlcrs/huawei_solar#279] [Setting up the integration] [answered]
i use the integration a few weeks. For testing i ´ve already deinstalled and reinstalled a few times 
today i update to latest 1.2.2 because the modbus issue
after update the integration works fine 
for testing modbus proxy (this always fails in my setup) wich  i´ve done with every new version since 1.1.4
i try do make a deinstall/install wich fails with unexpected error during setup

<img width="295" alt="Bildschirmfoto 2023-01-30 um 13 30 11" src="https://user-images.githubusercontent.com/85438367/215479397-ed2dcd3e-fd95-4e5b-a2cc-7a4f8f85fe5c.png">

when i make the reinstall with "no elevate permissions" checked - the installation works and the integration is running (without battery ...)
i checked this issue a few times to double check

Setup:
10KTL-M1 Master
- Power Sensor
- Luna2000 10kW
10KTL-M1 Slave

Edit: after downgrade to 1.2.1 - Integration works fine with elevate permissions

**Comment by wlcrs** [comment]
Please submit a proper bugreport including the logs containing the Modbus traffic. This will help me understand what is happening under the hood.

I would advise to generate a diagnostics file while you have the integration running without elevated permissions.

What is the latest version that was running fine for you?

Please also explain what you mean with Modbus proxy.

**Comment by scrichab** [comment]
in the moment i´m not @home so short answer

Home Assistant modbus proxy addon
- i just want to explain that i often deinstalled and reinstalled the integration to be able to change the modbus adress from direkt Inverter with Dongle Adress to modbus proxy Adress

latest running version 1.2.1

**Comment by scrichab** [comment, marked as answer]
works fine with version 1.2.3
thank you
